asdf/tests/test_api.py

```diff
@@ -234,6 +235,20 @@ def test_context_handler_resolve_and_inline(tmpdir):
         newf.tree['random'][0]
 
 
+def test_open_pathlib_path(tmpdir):
+
+    filename = str(tmpdir.join('pathlib.asdf'))
```

Comment:
Probably not relevant to this test, but `tmpdir` here is actually `pathlib.Path` like.

Reply:
Ah yeah, good point. I think I'm going to leave as-is to be explicit.